core/src/main/java/io/grpc/internal/BinaryLog.java

```diff
@@ -115,39 +133,44 @@ public static BinaryLog getLog(String fullMethodName) {
         String methodOrSvc = configMatcher.group(1);
         String binlogOptionStr = configMatcher.group(2);
         BinaryLog binLog = createBinaryLog(binlogOptionStr);
+        if (binLog == null) {
+          logger.log(Level.SEVERE, "Can not create binary log for: " + configuration);
```

Comment:
This will be a duplicate log entry, as `createBinaryLog()` also logs before returning `null`. Choose one.